android/libraries/rib-base/src/main/java/com/badoo/ribs/core/Node.kt

```diff
@@ -360,7 +337,7 @@ open class Node<V : RibView>(
         if (isAttachedToView) {
             ribLifecycleRegistry.markState(state)
             if (!isViewless) {
-                view!!.let { viewLifecycleRegistry.markState(state) }
+                view!!.let { viewLifecycleRegistry!!.markState(state) }
```

Comment:
why do you need `view!!.let` ?
just do `viewLifecycleRegistry.markState` maybe?

Reply:
good point, thanks.